lib/version.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: this should be sourced from package.json (maybe?)
```

Comment:
According to the [docs for `genversion`](https://www.npmjs.com/package/genversion), the whole purpose of this file is to _avoid_ bundling the `package.json`. Their main argument about the security implications isn't relevant to us, though, as this is an open source library, so the information is already available.

The only other consideration I can think of is whether it's worth including the whole large package.json just for the 5-character version.

Reply:
and if you consider using `tough-cookie` in the context of a browser or native environment then we may not have access to the `package.json` file.  

the only place this is actually leveraged is during serialization of cookies.  i'm not sure that's entirely necessary but that's a discussion for another day.  i've reintroduced the version check test that originally led to this TODO being noted.